src/components/services/user.ts

```diff
@@ -0,0 +1,5 @@
+import todos from '../../api/todos';
+
+export function getTodoById(id: number) {
+  return todos.filter(todo => todo.userId === id) || null;
```

Comment:
The use of `filter` will always return an array, even if it's empty, so the `|| null` part is unnecessary. If you want to return `null` when no todos are found, consider using `find` instead of `filter` if the intention is to find a single todo.